src/scenes/mainScene.ts

```diff
@@ -45,6 +62,23 @@ export class MainScene extends Scene {
             );
     }
 
+    private addPlayerMoneyText(player: IPlayer) {
+        this.playerMoneyText = new TextPlayerMoney(this, 50, 75, "", {});
+        this.children.add(this.playerMoneyText);
+        this.playerMoneyText.init(player);
+    }
+    private addBuyPriceColumn(city: ICity) {
```

Comment:
Consider moving the creation of `buyPriceText`s and `sellPriceText`s into `addRow(...)`